app/src/main/java/bluedot/com/au/bluedotbrazeintegrationapp/MainActivity.kt

```diff
@@ -26,10 +24,10 @@ class MainActivity : AppCompatActivity() {
             val app: MainApplication = this.application as MainApplication
 
             if (GeoTriggeringService.isRunning()) {
-                binding.bStartStopSDK.text = getString(R.string.stop_sdk)
+                binding.bStartStopSDK.text = getString(R.string.start_sdk)
```

Comment:
[Q/B] Do we need to change this to start? 
It looks it can cause the issue when the app will show Start button if the Geo is running and otherwise

Reply:
I updated the button text to allow user to perform the reverse action, Like here if user is pressing stop button and SDK is already running then it will attempt to stop the SDK and button will show Text as `Start SDK` which can be used to start the SDK again.

Reply:
I'm still not sure we need these changes. Let's have a call to discuss.

Reply:
As discussed will modify the UI to correct state of Geo-trigger later. merging the changes now.